src/index.js

```diff
@@ -1,30 +1,25 @@
-responseCount = 0
-currentQuestion = 0
+
 options = {
   url: 'data/quiz.json?' + Date.now()
 }
 
 $.ajax({
   url: options.url
 }).done(function(data) {
-  questions = data.questions
+  var responseCount = 0
+  var currentQuestion = 0
+  var responses = []
+  const {questions} = data
 
+  debugger
```

Comment:
Did the tests really passed with this ??!?!

Reply:
yes! I'll remove it!